letsdrill.rb

```diff
@@ -1,13 +1,22 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
-
+  grade = case integer
```

Comment:
Here's a few links to review.
https://github.com/paircolumbus/LetsDrill/pull/75

Keep this in your pocket too. It's a great guide.
https://github.com/bbatsov/ruby-style-guide#source-code-layout